privacyidea/lib/tokens/emailtoken.py

```diff
@@ -326,6 +328,16 @@ def _get_email_text_or_subject(options,
                 message = messages[0]
 
         message = message.format(challenge=options.get("challenge"))
+        if message.startswith("file:"):
+            # We read the template from the file.
+            try:
+                f = open(message[5:], "r")
+                message = f.read()
+                f.close()
```

Comment:
Minor nitpick: I think it would be cooler to use Python ``with`` (like we already do in some other places):
```python
with open(message[5:], "r") as f:
    message = f.read()
```

Reply:
right